integration/common.go

```diff
@@ -1,22 +1,22 @@
-package steelthread
+package integration
 
 import (
 	"bytes"
 	"embed"
+	"encoding/json"
+	cmpact "encoding/json"
```

Comment:
we should be using `"github.com/goccy/go-json"` as a drop in replacement for encoding/json everywhere

Reply:
my golang goes crazy and auto uses "encoding/json" even when I manually put in github.com/goccy/go-json,

it drives me crazy

Reply:
yeah it is super annoying. you could add a file watcher to auto replace https://www.jetbrains.com/help/go/settings-tools-file-watchers.html